### PR TITLE
[DOC] Explain how to integrate in Shiny Applications

### PR DESCRIPTION
**Pull request by tbouffard**
Based on https://github.com/process-analytics/bpmn-visualization-R/issues/23#issuecomment-909364915

covers #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,38 @@ overlays <- list(create_overlay("bpmn_element_id_1", "42"), create_overlay("bpmn
 bpmnVisualization::display(bpmn_file, overlays)
 ```
 
+
+### Integrate in Shinny Applications
+
+The following displays a BPMN diagram provided as example by the package with an overlay on top of a BPMN element
```
Comment (csouchet) [inline README.md:80]:
```suggestion
The following displays a BPMN diagram provided as example by the package with an overlay on top of a BPMN element.
```

File: README.md [hunk 2]
```diff
@@ -74,6 +74,38 @@ overlays <- list(create_overlay("bpmn_element_id_1", "42"), create_overlay("bpmn
...
+# Install and load packages
+install.packages("shiny")
+library(shiny)
+devtools::install_github("process-analytics/bpmn-visualization-R")
+library(bpmnVisualization)
+
+displayBpmn <- function() {
+    bpmn_file <- system.file("examples/Travel_Booking.bpmn", package = "bpmnVisualization")
+    overlays <- list(bpmnVisualization::create_overlay("_6-203", "9"))
+    bpmnVisualization::display(bpmn_file, overlays = overlays)
```
Comment (csouchet) [inline README.md:92]:
```suggestion
    bpmnVisualization::display(bpmn_file, overlays)
```

File: README.md [hunk 3]
```diff
@@ -74,6 +74,38 @@ overlays <- list(create_overlay("bpmn_element_id_1", "42"), create_overlay("bpmn
...
+    bpmnVisualizationOutput('bpmnContainer')
+  )
+)
+
+server = function(input, output) {
+    # renderBpmnVisualization is the R bridge function to the html widgets
+    output$bpmnContainer <- renderBpmnVisualization({ displayBpmn() })
+}
+
+shinyApp(ui = ui, server = server)
```
Comment (csouchet) [inline README.md:106]:
```suggestion
shinyApp(ui, server)
```

**Review by csouchet — CHANGES_REQUESTED** [review]

**Comment by csouchet** [comment]
Works fine 👍🏼 
image

File: README.md (hunk 3 above)
Comment (tbouffard) [reply README.md:106]: This line is taken from the shinny documentation, so I will leave it as it is.

Comment (csouchet) [reply README.md:106]: No need to put the variable name in R, like `ui =`, when the parameter are set in the defined order (in function) 😉 
http://adv-r.had.co.nz/Functions.html#function-arguments

I test without them, and it works. It avoid duplication.

Comment (tbouffard) [reply README.md:106]: I know that, but thanks for the link.
My remark was about being consistent with the usual way of integrating shinny code.
Anyway, I don't want to loose time on this, so I am applying your suggestion.

File: README.md [hunk 4]
```diff
@@ -74,6 +74,38 @@ overlays <- list(create_overlay("bpmn_element_id_1", "42"), create_overlay("bpmn
 bpmnVisualization::display(bpmn_file, overlays)
 ```
 
+
+### Integrate in Shinny Applications
```
Comment (tbouffard) [inline README.md:78]:
```suggestion
### Integrate in Shiny Applications
```

**Review by csouchet — APPROVED** [review]

**Pull request merged**